.github/workflows/publish.yml

```diff
@@ -0,0 +1,24 @@
+name: Publish new version
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  release:
+    if: "!contains(github.event.head_commit.message, 'skip ci')"
```

Comment:
I _think_ GitHub has native support for `skip ci` now, so you can likely exclude this.

Reply:
Okay cool, I pulled this from [these docs](https://github.com/BrightspaceUI/actions/tree/master/semantic-release#using-the-action) so if it's natively supported we can probably remove it from there.